### PR TITLE
Changed Material Dialog with Alert Dialog for ReadText.kt

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Replaces usages of MaterialDialog with AlertDialog in ReadText.kt
## Fixes
Related to #13315 

## How Has This Been Tested?
Open each one of the changed dialogs on their respective preferences

### CODE REVIEW

**Review by david-allison — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/ReadText.kt [hunk 1]
```diff
@@ -97,7 +99,7 @@ object ReadText {
         mDid = did
         mOrd = ord
         val res = flashCardViewer.get()!!.resources
-        val dialog = MaterialDialog(flashCardViewer.get()!!)
+        var dialog = AlertDialog.Builder(flashCardViewer.get()!!)
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/ReadText.kt:102]: Please don't introduce unnecessary changes like `val` to `var`.

**Review by lukstbit — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/ReadText.kt (hunk 1 above)
Comment (Prince-kushwaha) [reply AnkiDroid/src/main/java/com/ichi2/anki/ReadText.kt]: ok it is fixed

**Review by lukstbit — APPROVED** [review]
Thanks!
